### DISCUSSION TITLE
Long term sustainability

### DISCUSSION THREAD

**Discussion by erlend-sh** [AppFlowy-IO/AppFlowy#15] [General]
The about page doesn’t say anything about the long term aspirations of the project, I.e. how it will be sustainable. Are you planning to turn this into a business? I hope so!

**Comment by annieappflowy** [comment]
Hi Erlend,

Thanks for checking us out! We'd like to make it our long-term career and create a lasting positive impact through it along with the community, so a ramen profitable business model is something we need to come up with in order to make it sustainable. Any ideas or thoughts are welcomed!

Annie

**Reply by Lloyd-Jackman-UKPL** [reply]
I'd be willing to chip in some time for non coding work, but appreciate that you guys will need money to scale it. I know I'm usually willing to pay up $5 a month for something I'll use without thinking about it, but making that recurring without paying a host of fees is clearly where you need to watch (ahem PayPal). 

As something to entice folk, leveraging the template community kind of thing that Notion has done well with could be an idea. Something like your $5 recurring gives access to one free template a month. For more than that, you'd pay something to the author of the template. As for the free one given, the author gets a cut (say $1) and their subscription covered for another month. Happy to try and map something out if what I'm getting at isn't clear but the aim is to drive the use of templates and therefore build in user value quickly, whilst rewarding a community of template authors.

**Reply by pesho-ivanov** [reply]
@annieappflowy, is there a particular reason you are not running a hosting for it? This looks like the obvious missing feature for most users (incl. me). Is there any paid hosting someone provides?

**Reply by annieappflowy** [reply]
We're working on cloud solutions. Likely by July, you will be able to self-host or subscribe to our cloud service.
